### ISSUE TITLE
Web Bundle support

### ISSUE THREAD

**Issue by FluorescentHallucinogen** [pwa-builder/PWABuilder-CLI#340]
I see Web Bundles (`*.wbn` files) as universal executable format for web apps in the future (instead of `*.apk` (WebAPK, TWA), `*.appx`, etc.).

Read more about Web Bundles here: https://web.dev/web-bundles/.

Currently, PWABuilder can output APK (Trusted Web Activity) files.

TWA has a problem that if the device is offline in the first run, TWA can't show any content. Also, it takes time to load the page in the first run on flaky network.

Looks like [Web Bundles](https://web.dev/web-bundles/) can fix this.

1. What about generating the `*.wbn` file and placing it inside APK?

TWA can open the web bundle file from the APK in the first run (the page in the bundle file installs a service worker).

Web Bundles can be created using [wbn](https://www.npmjs.com/package/wbn) npm package.

2. What about adding support of `*.wbn` files to PWABuilder as one of the output format?

3. What about adding support of `*.wbn` files to Microsoft Store as one of the input format?

The web bundle file contains all necessary data to publish in app stores: web app manifest with name, icons, description and screenshots.

It's possible to verify the owner and integrity of signed web bundles.

**Comment by Jaifroid** [comment]
But it only works on Chrome?

**Comment by FluorescentHallucinogen** [comment]
@Jaifroid Yes, at this moment it's implemented only in Chromium. So it's expected in the near future in other Chromium-based browsers like Microsoft Edge, Samsung Internet, etc.

**Comment by getify** [comment]
I would like this as well!
